packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -57,5 +54,5 @@
   "force_source_delete_this_source_text": "SFDX: Delete This from Project and Org",
   "force_config_set_org_text": "SFDX: Set a Default Org",
   "internal_dev_description": "Specifies whether or not to set up internal development mode which provides limited SFDX commands.",
-  "refresh_org_browser_text": "Refresh Org Browser"
```

Comment:
Need opinions here. I wanted to have a separate label for the individual refresh buttons from the global refresh, but that would involve adding another command just to have the different label. Is it okay if I generalize it more to "Refresh"?